src/test/kotlin/test/assertk/assertions/PathSpec.kt

```diff
@@ -0,0 +1,116 @@
+package test.assertk.assertions
+
+import assertk.assert
+import assertk.assertions.*
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.given
+import org.jetbrains.spek.api.dsl.it
+import java.nio.file.Files
+import java.nio.file.Path
+
+class PathSpec : Spek({
+    var regularFile: Path? = null
```

Comment:
You can use `lateinit` to not have to `!!` everywhere.

Reply:
AFAIK lateinit works only for members. Only starting with kotlin 1.2 you can put into methods.